hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/jobhistory/TestHistoryViewerPrinter.java

```diff
@@ -43,6 +46,19 @@
 
   private final String LINE_SEPARATOR = System.lineSeparator();
 
+  private Locale defaultLocale;
+
+  @Before
```

Comment:
This can be `@BeforeClass`? Same to `tearDown` to be static.

Other than that +1

Reply:
@liuml07 Okay, I will push that changes.